pkg/controller/build/ocl_events.go

```
@@ -21,11 +22,11 @@ const (
 	EventBuildDeleted     = "BuildDeleted"
 
 	// Job events
-	EventJobCreated   = "JobCreated"
-	EventJobStarted   = "JobStarted"
-	EventJobCompleted = "JobCompleted"
-	EventJobFailed    = "JobFailed"
-	EventJobDeleted   = "JobDeleted"
+	EventJobCreated    = "JobCreated"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)